src/shipit_static_analysis/shipit_static_analysis/cli.py

```diff
@@ -63,33 +68,44 @@ def main(commits,
                 MOZDEF=secrets.get('MOZDEF'),
                 )
 
-    mozreview = build_api_root(
-        secrets['MOZREVIEW_URL'],
-        secrets['MOZREVIEW_USER'],
-        secrets['MOZREVIEW_API_KEY'],
+    # Load reporters
+    reporters = get_reporters(
+        secrets['REPORTERS'],
+        taskcluster_client_id,
+        taskcluster_access_token,
     )
 
+    # Load revisions
+    revisions = []
+    if phabricator:
+        # Only one phabricator revision at a time
+        api = reporters.get('phabricator')
+        assert api is not None, \
+            'Cannot use a phabricator revision without a phabricator reporter'
+        revisions.append(PhabricatorRevision(phabricator, api))
+    if mozreview:
+        # Multiple mozreview revisions are possible
```

Comment:
Interesting. I forgot, why do we only process one Phabricator revision at a time, but multiple MozReview revisions are possible?

Reply:
* Mozreview pulse messages *can* embed multiple revisions in a single message. Even if they are not about the same commit nor review. So we can have multople revisions per TC task
 * Phabricator feed lists unique diffs, so i create only one revision per TC task

Some mozreview TC tasks end up being too long and get killed by the watchdog. So it would be better to limit the static analysis bot to run only one review per task (let's rely more on TC queue)